includes/polling/ports.inc.php

```diff
@@ -274,8 +288,41 @@
 }//end if
 
 if ($config['enable_ports_poe']) {
-    $port_stats = snmpwalk_cache_oid($device, 'pethPsePortEntry', $port_stats, 'POWER-ETHERNET-MIB');
```

Comment:
With your new code you've now stopped this working for any none ios, procurve or vrp devices, how come?

Reply:
The existing code was not working (POWER-ETHERNET-MIB is polled but never used in the code, and CISCO-POWER-ETHERNET-EXT-MIB is CISCO specific, but in case it was used by somebody, I restored the original code in the final 'else' statement now.